example/lib/draggable_sample.dart

```diff
@@ -0,0 +1,62 @@
+import 'package:flame_forge2d/flame_forge2d.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:forge2d/forge2d.dart';
+import 'package:flame_forge2d/forge2d_game.dart';
+import 'package:flame/palette.dart';
+import 'package:flame/gestures.dart';
+import 'package:flame/extensions/offset.dart';
+import 'package:flame/components/mixins/dragable.dart';
+
+import 'balls.dart';
+import 'boundaries.dart';
+
+class DraggableSample extends Forge2DGame with HasDragableComponents {
+  DraggableSample(Vector2 viewportSize)
+      : super(
+          scale: 4.0,
+          gravity: Vector2(0, 0.0),
+        ) {
+    viewport.resize(viewportSize);
+    final boundaries = createBoundaries(viewport);
+    boundaries.forEach(add);
+    add(DraggableBall(viewport.size / 2));
+  }
+
+  @override
+  void onReceiveDrag(DragEvent event) {
```

Comment:
Is it necessary to override this, it is just calling super

Reply:
It is not, I was just troubleshooting why it doesn't receive the initial drag in the correct position.
Then I realized it is because of the sidebar in dashbook, so trying to figure out how to set the initialPosition correctly in flame.